eventbus/jetstream/sensor/trigger_conn.go

```diff
@@ -257,15 +257,35 @@ func (conn *JetstreamTriggerConn) processMsg(
 		conn.Logger.Errorf("can't get Metadata() for message %+v??", m)
 	}
 
-	defer func() {
-		err = m.AckSync() // this call does the acknowledgment and waits for an acknowledgment from the server (that it received the ack) too
-		if err != nil {
-			errStr := fmt.Sprintf("Error performing AckSync() on message: %v", err)
-			conn.Logger.Error(errStr)
+	done := make(chan bool)
+	go func() {
+		ticker := time.NewTicker(500 * time.Millisecond)
```

Comment:
500ms is too long

Reply:
It looks like we don't set an `AckWait` when we create the [pull subscription](https://github.com/argoproj/argo-events/blob/master/eventbus/jetstream/sensor/trigger_conn.go#L136). Do you know what the default is? I can't find it in the docs [here](https://pkg.go.dev/github.com/nats-io/nats.go#AckWait)